torch/testing/_internal/common_methods_invocations.py

```diff
@@ -15625,7 +15679,21 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                # Computed gradient is incorrect -- would be an exfail but gradgrad somehow passes
                DecorateInfo(unittest.skip("Gradients are incorrect!"), 'TestFwdGradients'),
                DecorateInfo(unittest.skip("Gradients are incorrect!"), 'TestBwdGradients'),
-               DecorateInfo(unittest.skip('output is non-deterministic'), 'TestCommon', 'test_compare_cpu'))),
+               DecorateInfo(unittest.skip('output is non-deterministic'), 'TestCommon', 'test_compare_cpu'),
+               # The inplace variant (Tensor.normal_) is different from torch.normal
```

Comment:
Hi @lezcano, could you help have another look at the opinfo since the last approval ? I added skipping some of the original tests (that passed without the normal decomp) in other two existing variants of `normal` opinfo.